pallets/circuit/src/lib.rs

```diff
@@ -332,7 +370,7 @@ pub mod pallet {
             // Anything that needs to be done at the start of the block.
             // We don't do anything here.
             // ToDo: Do active xtx signals overview and Cancel if time elapsed
-            0
+            weight
```

Comment:
create a ticket to update weight after processing the timeouts queue

Reply:
https://github.com/t3rn/t3rn/issues/345